### ISSUE TITLE
`pyqtgraph.PlotItem` is incorrectly resolved as a module instead of a class

### ISSUE THREAD

**Issue by t-onoz** [microsoft/pyright#11641]
**Describe the bug**

Pyright resolves `pyqtgraph.PlotItem` as a module, but it is a class at runtime.

I would expect Pyright to resolve `pg.PlotItem` as the `PlotItem` class.

**Code or Screenshots**

`test_pg.py`:
```python
import pyqtgraph as pg
from typing import TYPE_CHECKING

print(pg.PlotItem)
print(type(pg.PlotItem))

if TYPE_CHECKING:
    reveal_type(pg.PlotItem)

def foo(plot_item: pg.PlotItem) -> None:
    pass
```

Runtime output (`python test_pg.py`):

```text
<class 'pyqtgraph.graphicsItems.PlotItem.PlotItem.PlotItem'>
<class 'PyQt6.sip.wrappertype'>
```

Pyright output (`pyright test_pg.py`):

```text
test_pg.py:8:17 - information: Type of "pg.PlotItem" is "Module(".PlotItem")"
test_pg.py:10:20 - error: Module cannot be used as a type (reportGeneralTypeIssues)
1 error, 0 warnings, 1 information
```

`PlotItem` is re-exported by pyqtgraph.

In `pyqtgraph/__init__.py`:

```python
from .graphicsItems.PlotItem import *
```

In `pyqtgraph/graphicsItems/PlotItem/__init__.py`:

```python
from .PlotItem import PlotItem

__all__ = ["PlotItem"]
```

pyqtgraph version: `0.14.0`

**VS Code extension or command-line**

Command-line Pyright `1.1.411`.

**Comment by t-onoz** [comment]
This appears to be a duplicate of #11481. Closing in favor of that issue.
